### PR TITLE
Made the `hic` param pattern more flexible

### PR DESCRIPTION
**Pull request by GallVp**
## PR checklist

- [x] Made the `hic` param pattern more flexible
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/plant-food-research-open/assemblyqc/tree/main/.github/CONTRIBUTING.md)
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by GallVp** [comment]
For visibility, @SarahBailey1998, @ignacio3437

File: tests/hicparam/hicparam.nf.test [hunk 1]
```diff
@@ -0,0 +1,118 @@
+nextflow_function {
+
+    name "checkHiCParam"
+    script "./hicparam.nf"
+    function "checkHiCParam"
+
+    test("Dummy_hic.R{1,2}.fq.gz") {
```
Comment (GallVp) [inline tests/hicparam/hicparam.nf.test:7]: @SarahBailey1998, @ignacio3437

The permissible patterns are tested here. Are we missing something?

Comment (SarahBailey1998) [reply tests/hicparam/hicparam.nf.test:7]: would "dummy_hic_{1,2).merged.fastq.gz be allowed?

Comment (GallVp) [reply tests/hicparam/hicparam.nf.test:7]: If you mean: `dummy_hic_{1,2}.merged.fastq.gz` and not `dummy_hic_{1,2).merged.fastq.gz`, then yes.

The permissible pattern is `^\S+\{1,2\}[\w\.]*\.f(ast)?q\.gz$`

\S+ will cover the dummy_hic_ part

We can test it here: https://regex101.com

Comment (SarahBailey1998) [reply tests/hicparam/hicparam.nf.test:7]: yeah that's what I mean, I made a typo sorry! Awesome

**Pull request merged**